scripts/setup-volume-testnet.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+
+if ! which jq > /dev/null; then
+  echo 'command jq not found, please install jq'
+  exit 1
+fi
+
+CHAIN_ID="paloma"
+VALIDATOR_STAKE_AMOUNT=100000000grain
+
+palomad init my_validator --chain-id "$CHAIN_ID"
+
+pushd ~/.paloma/config/
+sed -i 's/^keyring-backend = ".*"/keyring-backend = "test"/' client.toml
+sed -i 's/^minimum-gas-prices = ".*"/minimum-gas-prices = "0.001grain"/' app.toml
+sed -i 's/^laddr = ".*:26657"/laddr = "tcp:\/\/0.0.0.0:26657"/' config.toml
+jq ".chain_id = \"${CHAIN_ID}\"" genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+jq 'walk(if type == "string" and .. == "stake" then "grain" else . end)' genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+popd
+
+name="chase"
+echo "$MNEMONIC" | palomad keys add "$name" --recover
+address="$(palomad keys show "$name" -a)"
+
+palomad add-genesis-account "$address" 4000000000grain
```

Comment:
@verabehr As written this account has 4Ggr, the below accounts get 800Mgr total. What would you like the distribution to be?

Reply:
ok, so let's change the faucet to give out 100 000grain per request. 
us for (jason, taariq, you, me) each get 250000000 as you have it. 
the faucet starts out with 500 000 000grain as you have it
that gives us 1 500 000 000grain in total.

Reply:
Done.